tests/e2e/cucumber/features/smoke/spaces/publicLink.ocis.feature

```diff
@@ -23,11 +23,11 @@ Feature: spaces public link
             | spaceFolder | folder |
         And "Alice" creates a public link for the resource "spaceFolder" using the sidebar panel
         And "Alice" renames the most recently created public link of resource "spaceFolder" to "folderLink"
-        And "Alice" adds following users to the project space
-            | user  | role    |
-            | Brian | editor  |
-            | Carol | viewer  |
-            | Marie | manager |
+        And "Alice" adds following members to the project space
+            | member | role    |
```

Comment:
should probably add the `kind` column? otherwise the test will search for the `Brian` group